pkg/pubsub/adapter/converters/storage.go

```diff
@@ -88,7 +88,8 @@ func convertStorage(ctx context.Context, msg *cepubsub.Message, sendMode ModeTyp
 		}
 	}
 	event.SetDataContentType(*cloudevents.StringOfApplicationJSON())
-	event.SetData(msg.Data)
+	event.Data = msg.Data
```

Comment:
what was the issue here? why did you have to directly set?

Reply:
I see in https://github.com/google/knative-gcp/blob/master/pkg/pubsub/adapter/converters/pubsub.go#L44-L45, it directly sets the `Data`. I'm not sure if it's intended to be different. For unit tests here, it only affects how I construct the wanted event.

Reply:
ah, this in in convert. +1